src/store/mod.rs

```diff
@@ -39,18 +39,28 @@ mod writer;
 pub use self::reader::StoreReader;
 pub use self::writer::StoreWriter;
 
+#[cfg(all(feature = "lz4", feature = "brotli"))]
+compile_error!("feature `lz4` or `brotli` shouldn't be enabled both.");
```

Comment:
... "must not be enabled together."

- I am not a native speaker, but I think together is slightly more correct english -unsure I make a load of grammar mistakes-  
- rfc2119 describes what should be our definition of MAY, SHOULD, MUST, SHOULD NOT, MUST NOT in documetnation